packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -2135,11 +2135,23 @@ export const audiusBackend = ({
 
   /**
    * Sets the artist pick for a user
+   * @param {User} userMetadata
+   * @param {number} userId
    * @param {number?} trackId if null, unsets the artist pick
    */
-  async function setArtistPick(trackId: Nullable<ID> = null) {
+  async function setArtistPick(
+    userMetadata: User,
+    userId: ID,
+    trackId: Nullable<ID> = null
+  ) {
     await waitForLibsInit()
     try {
+      // Dual write to the artist_pick_track_id field in the
+      // users table in the discovery DB. Part of the migration
+      // of the artist pick feature from the identity service
+      // to the entity manager in discovery.
+      updateCreator(userMetadata, userId)
```

Comment:
`updateCreator` in libs is gated by the user entity manager flag. The user metadata update logic is the same as the old flow but there is all the new encapsulating entity manager logic in `entity_manager.py` that will be run so it seems risky to not gate this with the user entity manager flag @jowlee

Reply:
It shouldn't be gated on entity manager flag. There is a params `useEntityManager` that's passed to it which toggles the flow to the update to the user's metadata from the current User contract to being doing with the entity manager contract. 
On the discovery indexing side, it be calling the same method to update the user - but I do agree that it's stacking changes that could lead to a bug. 
We could wait to roll out until entity manager it out